### ISSUE TITLE
Hyperion not always started with a good default state

### ISSUE THREAD

**Issue by DominicOram** [DiamondLightSource/mx-bluesky#484]
When a user does the following:
1. Unload sample
2. Prepare for access
3. Run Hyperion

They get an error of `dodal.devices.aperturescatterguard.InvalidApertureMove: ApertureScatterguard safe move is not yet defined for positions outside of LARGE, MEDIUM, SMALL, ROBOT_LOAD. Current aperture z (-10.0009), outside of tolerance (0.1) from target (15.8).`

This implies that the beamline is not always being put in a good default state by GDA.

## Acceptance Criteria
- The above workflow works without error

**Comment by rtuck99** [comment]
After some investigation into GDA, it seems there are three places where initial checks are performed:

* Depending on how data collection is invoked, there are different code paths followed before `HandleCollectRequests.execute()` is invoked:
   * If it is invoked via UDC, then `UDCRunner` invokes `setBeamlineToDefaultState()` which then runs `beamline_default_sate.set_to_default_state()` in mx-config scripts
   * If it is invoked via the UI then `DataCollectionScanManager.runDataCollectionScan()` does some of its own checks
* Then there is some common checking done in `CollectionRequestHandler`

**Comment by DominicOram** [comment]
We are now getting an error on the beamstop not being in instead.

**Comment by DominicOram** [comment]
This error was because we were previously doing a `context.aperture_scatterguard_scintillator.moveToRobotLoad()` before every collection. This moves most things into the correct position but does not move the backstop in correctly. We now do:
```
# Need to make sure that we've safely moved the aperture/scatterguard out from under the table
context.aperture_scatterguard_scintillator.moveToRobotLoad()
# If we have come from a safe hutch moveToRobotLoad will not put the backstop in so do this
context.aperture_scatterguard_scintillator.moveBackstopToDataCollectionPos()
```
for every collection. This moves everything in to the correct position if we're coming from a safe hutch but is a no-op if we've just finished a collection

**Comment by olliesilvester** [comment]
Code looks good, but before closing can we confirm that the 
>
>    1. Unload sample
>    2. Prepare for access
>    3. Run Hyperion
>

works without errors now?

**Comment by rtuck99** [comment]
Changes look good - this is essentially what the non-hyperion code path was doing in `GDARobotLoadAndCentreFunctions.execute_sample_loading()`

**Comment by DominicOram** [comment]
> Code looks good, but before closing can we confirm that the
> 
> > 1. Unload sample
> > 2. Prepare for access
> > 3. Run Hyperion
> 
> works without errors now?

Yes, this worked on the beamline when I was last down there. @npat113 have you seen any more errors after prepare for access then running hyperion?
